src/test/java/chess/domain/dao/PieceDaoTest.java

```diff
@@ -0,0 +1,53 @@
+package chess.domain.dao;
+
+import static chess.domain.chesspiece.Team.BLACK;
+import static chess.domain.chesspiece.Team.WHITE;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.junit.jupiter.api.Assertions.*;
+
+import chess.domain.GameStatus;
+import chess.domain.chesspiece.Piece;
+import chess.domain.chesspiece.slidingPiece.King;
+import java.sql.Connection;
+import java.sql.SQLException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class PieceDaoTest {
+    private PieceDao pieceDao;
+    private static Connection connection;
+
+    @BeforeAll
+    static void connection() {
+        try {
+            connection = ConnectionGenerator.getConnection();
+            connection.setAutoCommit(false);
+        } catch(SQLException ignored) {
+        }
+    }
```

Comment:
```suggestion
    static void connection() throws SQLException {
        connection = ConnectionGenerator.getConnection();
        connection.setAutoCommit(false);
    }
```
1. 테스트메소드에서는 try-catch로 굳이 감싸주지 않으셔도 될 것 같아요.
2. connection을 다 쓴 이후에는 setAutoCommit을 true로 바꾸어줘야하지 않나요?